chain/chain/src/stateless_validation/mod.rs

```diff
@@ -1,2 +1,3 @@
 pub(crate) mod chunk_endorsement;
 pub(crate) mod state_transition_data;
+pub mod state_witness;
```

Comment:
it doesn't seem like a descriptive module name, I find it rather confusing 🙃 
I suggest changing it to `chunk_validation`